src/common/docker-compose/index.ts

```diff
@@ -31,7 +31,7 @@ export const generate = async (dependency_manager: LocalDependencyManager): Prom
   for (const node of dependency_manager.graph.nodes) {
     if (node instanceof GatewayNode) {
       compose.services[node.normalized_ref] = {
-        image: 'jwilder/nginx-proxy',
+        image: 'registry.architect.io/architect-nginx/proxy:latest',
         restart: 'always',
         ports: [`${await dependency_manager.gateway_port}:${node.ports[0]}`],
```

Comment:
Can you add the env var for DISABLE_ACCESS_LOGS?

Reply:
I added it to the Dockerfile when I built the image from source so that we didn't need to keep it here. Did you want it here instead? Or should we keep a copy of the Dockerfile somewhere? it's a one line change

Reply:
Yeah I don't want to fork their docker image. Lets build their master's dockerfile and add the env here.